react/fields/Dropdown/Dropdown.js

```diff
@@ -135,12 +135,15 @@ export default class Dropdown extends Component {
       [className]: className
     });
 
+    // eslint-disable-next-line react/prop-types
+    const { id, label, labelProps, secondaryLabel, invalid, help, helpProps, message, messageProps } = this.props;
+
     return (
       <div className={classNames}>
-        <FieldLabel {...this.props} />
+        <FieldLabel id={id} label={label} labelProps={labelProps} secondaryLabel={secondaryLabel} />
```

Comment:
I'd write this as `<FieldLabel {...{ id, label, labelProps, secondaryLabel }} />`